pom.xml

```diff
@@ -295,7 +295,7 @@
     <dependency>
       <groupId>com.google.googlejavaformat</groupId>
       <artifactId>google-java-format</artifactId>
-      <version>1.7</version>
+      <version>1.14.0</version>
```

Comment:
Since 1.8, it requires JDK 11.

> The minimum supported runtime version is now JDK 11.

https://github.com/google/google-java-format/releases/tag/google-java-format-1.8